CHANGELOG.adoc

```diff
@@ -16,6 +16,8 @@ For a detailed view of what has changed, refer to the {url-repo}/commits/master[
 Improvement::
 
 * Allow defining `@Name` as a meta annotation on Block and Inline Macros (@uniqueck) (#898)
+* Upgrade to jruby 9.2.17.0 (#1004)
+* Upgrade to asciidoctorj-diagram 2.1.0 (#1004)
```

Comment:
`2.1.0` does not match the one below `2.1.2` 🙄

Reply:
Thanks for the pointer, stupid me... 😖
I'll fix this asap.